creator-node/src/routes/files.js

```diff
@@ -247,46 +252,89 @@ module.exports = function (app) {
     // Do not act as a public gateway. Only serve IPFS files that are tracked by this creator node.
     const dirCID = req.params.dirCID
     const filename = req.params.filename
+    const ipfsPath = `${dirCID}/${filename}`
 
-    const queryResults = await models.File.findOne({ where: {
-      multihash: dirCID,
-      type: 'dir'
-    } })
+    const queryResults = await models.File.findOne({
+      where: {
+        storagePath: { [models.Sequelize.Op.like]: `%${dirCID}%` },
+        // Could look like blob/150x150.jpg or artwork/150x150.jpg
+        sourceFile: { [models.Sequelize.Op.like]: `%${filename}` },
+        type: { [models.Sequelize.Op.ne]: 'dir' }
```

Comment:
same question from previous PR - why allow all non-dir types, instead of just images?

Reply:
There's no reason to block this route on images only because it's not

/image/CID/size.jpg, it's just /CID/size.jpg. If we ended up putting anything else in a dir (not saying we would necessarily), this breaks?

We don't even need this part of it if we added the sep. columns / add an index because we can just query directly.

Also -- I want this endpoint to be as similar as the above one, which does this same thing

Reply:
i guess, i don't see a reason to open this up now in anticipation of future dir usage - rather, if and when we use dirs for other file types, we can modify route at that point

my concern is simply that we shouldn't open this up for no reason without testing it on every file type / edge case. it probably works, but what's the value? clients don't use it, and have no need to use it since the above route fulfills that. just seems a change that adds no value with potential risk, esp when this PR is for a diff task.

if you've tested every file type thoroughly with this route, then good with me. if not, seems just unnecessary?